src/app/body/task-details/task-details.component.html

```diff
@@ -27,116 +32,127 @@ <h5 class="active nav-link navbar-text" style="font-weight:bolder">Task Details<
                     </div>
                 </nav>
                 <div class="card-body">
-                    <div class="row">
-                        <div class="alert form-inline w-50 col-md-4 col-5 mb-5" style="font-size: 20px;" role="alert">
-                            <div style="font-weight: bold;" class="mr-3">Date :</div>
-                            <div id="creationDateTaskDescription">{{task.CreationDate}}</div>
-                        </div>
-                        <div class="col-md-4 col-1"></div>
-                        <div class="alert form-inline col-md-4 col-5 mb-5" style="font-size: 20px; background-color: #10be65;color:black" role="alert" *ngIf="task.Status == 'Completed'">
-                            <div style="font-weight: 500;" class="mr-3">Completion Date :</div>
-                            <div id="creationDateTaskDescription">{{task.CompletionDate}}</div>
-                        </div>
```

Comment:
you have removed the completion date. Please add it

Reply:
ok will do the needful.